apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -115,13 +116,47 @@ const SuspendableSiteList = (): JSX.Element => {
       <Flex flexDirection="column" gap="2rem">
         <SimpleGrid columns={3} gap="2.5rem" width="100%">
           {sites.map((site) => (
-            <Card key={site.id} width="100%">
-              <CardHeader>
-                <Link href={`/sites/${site.id}`} as={NextLink}>
-                  {site.name}
-                </Link>
-              </CardHeader>
-            </Card>
+            <Link
```

Comment:
i haven't tried locally, but i know that there's a pitfall which chakra solves [here](https://v2.chakra-ui.com/docs/components/link-overlay)

Reply:
abit lazy of me but can you check!

Reply:
oooo nice, thanks!